feat(curriculum): add greeting card workshop to cert (#56976)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-greeting-card/67332e9debcc820d18e3e169.md

---
id: 67332e9debcc820d18e3e169
title: Step 12
challengeType: 0
dashedName: step-12
---

# --description--

> Add a new selector that targets the `.card` element when hovered to change it's `background-color` to `khaki`.

Reviewer: ```suggestion
Add a new selector that changes the `background-color` of the `.card` element to `khaki` when it is hovered over.
```
